### PR TITLE
[Filter/Caffe2] fix invalid data type

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. fix invalid tensor meta while running invoke callback
2. add debug messages for tensor meta info

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc [hunk 1]
```diff
@@ -262,11 +263,17 @@ Caffe2Core::run (const GstTensorMemory * input, GstTensorMemory * output)
...
+  for (i = 0; i < inputTensorMeta.num_tensors; i++) {
     Tensor *inputTensor = inputTensorMap.
                             find(inputTensorMeta.info[i].name)->second;
-    inputTensor->ShareExternalPointer ((float*) input[i].data);
-    switch (outputTensorMeta.info[i].type){
+
+#if (DBG)
+    int num_dims = inputTensor->dim();
+    g_message ("input[%d] dims %d", i, num_dims);
+#endif
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: This looks like temporary debug code to be removed?

File: ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc [hunk 2]
```diff
@@ -262,11 +263,17 @@ Caffe2Core::run (const GstTensorMemory * input, GstTensorMemory * output)
 #if (DBG)
   gint64 start_time = g_get_real_time ();
 #endif
-  for (i = 0; i < inputTensorMeta.num_tensors; i++){
+
+  for (i = 0; i < inputTensorMeta.num_tensors; i++) {
     Tensor *inputTensor = inputTensorMap.
                             find(inputTensorMeta.info[i].name)->second;
-    inputTensor->ShareExternalPointer ((float*) input[i].data);
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: Is this the only effective change of this PR?

Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: line 276 is major change (in switch cond, output > input)

File: ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc (hunk 1 above)
Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.cc]: I will remove the debug messages

**Pull request merged**